examples/generate.sh

```diff
@@ -27,8 +27,9 @@ command -v "${ENVSUBST}" >/dev/null 2>&1 || echo -v "Cannot find ${ENVSUBST} in
 CLUSTERAWSADM=${CLUSTERAWSADM:-clusterawsadm}
 command -v "${CLUSTERAWSADM}" >/dev/null 2>&1 || echo -v "Cannot find ${CLUSTERAWSADM} in path, build it using 'make binaries' in this repository."
 
-# Cluster name.
+# Cluster.
 export CLUSTER_NAME="${CLUSTER_NAME:-test1}"
+export KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.14.4}"
```

Comment:
We should probably update the other existing templates for the k8s version env variable as well.